### PR TITLE
Adding jobs for bosh environment setup specific for PCF installation

### PR DESCRIPTION
**Pull request by keerthibokka**

### CODE REVIEW

File: jobs/consul-controller-dns-setup/spec [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (amittbose) [inline jobs/consul-controller-dns-setup/spec]: don't you need an empty "monit" file for this job as well?

File: jobs/consul-dns-setup/spec [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: consul-dns-setup
```
Comment (amittbose) [inline jobs/consul-dns-setup/spec]: Rename this job to "consul-gorouter-dns-setup"

File: jobs/consul-controller-dns-setup/spec (hunk 1 above)
Comment (keerthibokka) [reply jobs/consul-controller-dns-setup/spec]: jobs/consul-controller-dns-setup/monit is present in the pull request.

File: jobs/consul-dns-setup/spec (hunk 2 above)
Comment (keerthibokka) [reply jobs/consul-dns-setup/spec]: Will change job same as suggested.

**Pull request closed without merging**